src/pykeen/nn/representation.py

```diff
@@ -347,7 +347,7 @@ def __init__(
         # work-around until full complex support (torch==1.10 still does not work)
         # TODO: verify that this is our understanding of complex!
         if dtype.is_complex:
-            shape = tuple(shape[:-1]) + (2 * shape[-1],)
+            shape = tuple(shape[:-1]) + (shape[-1], 2)
```

Comment:
this is the shape [torch.view_as_complex](https://pytorch.org/docs/stable/generated/torch.view_as_complex.html) expects, i.e., `(..., 2)`